iron-overlay-manager.html

```diff
@@ -206,6 +173,56 @@
     return z;
   };
 
+  Polymer.IronOverlayManagerClass.prototype._setZ = function(element, z) {
+    element.style.zIndex = z;
+  };
+
+  Polymer.IronOverlayManagerClass.prototype._applyOverlayZ = function(overlay, aboveZ) {
+    this._setZ(overlay, aboveZ + 2);
+  };
+
+  /**
+   * Returns the overlay containing the provided node. If the node is an overlay,
+   * it returns the node.
+   * @private
+   * @param {Node} node
+   * @returns {?Node} overlay
+   */
+  Polymer.IronOverlayManagerClass.prototype._overlayParent = function(node) {
+    while (node && node !== document.body) {
+      // Check if it is an overlay.
+      if (node.behaviors && node.behaviors.indexOf(Polymer.IronOverlayBehaviorImpl) !== -1) {
+        return node;
+      }
+      // Use logical parentNode, or native ShadowRoot host.
+      node = Polymer.dom(node).parentNode || node.host;
+    }
+  };
+
+  /**
+   * @private
+   * Ensures the click event is delegated to the right overlay.
+   */
+  Polymer.IronOverlayManagerClass.prototype._onCaptureClick = function(event) {
+    var overlay = this.currentOverlay();
+    // Check if clicked outside of any overlay.
+    var target = Polymer.dom(event).rootTarget;
+    if (overlay && this._overlayParent(target) !== overlay) {
+      overlay._onCaptureClick(event);
```

Comment:
@cdata, to implement the behavior you were suggesting (aka, one click outside the overlay should not reach the other elements), it would be enough to call `event.stopPropagation()` in here